spdk.spec

```diff
@@ -139,7 +139,8 @@ BuildArch: noarch
             --without-iscsi-initiator \
             --without-isal \
             --without-vtune \
-            --with-shared
+            --with-shared \
+            --enable-debug
```

Comment:
Just curious, Is this going to have any performance impact?

Reply:
yes , not planning to land, it's just to enable SPDK VMD module debug

Reply:
to help with DAOS-9972 debugging